pkg/networkservice/chains/nsmgr/heal_test.go

```diff
@@ -113,65 +118,57 @@ func testNSMGRHealEndpoint(t *testing.T, restored bool) {
 }
 
 func TestNSMGR_HealLocalForwarder(t *testing.T) {
-	t.Skip("https://github.com/networkservicemesh/sdk/issues/845")
-
-	forwarderCtx, forwarderCtxCancel := context.WithTimeout(context.Background(), time.Second)
+	forwarderCtx, forwarderCtxCancel := context.WithCancel(context.Background())
```

Comment:
Why do we need to remove test timeouts from the root context?

Reply:
Because we manually close the hop to heal when we need it. We have a global timeout for the test, so it wouldn't last forever - this timeout makes no sense at all.

Reply:
The current global timeout is 5m.
We expect that sandbox test should pass during 5s.
So current heal tests could pass in 2-3min and be valid when it is not.

Reply:
My fault, it is not quite clear, but I have meant this as a _global timeout_:
https://github.com/networkservicemesh/sdk/blob/c06e6f43c17c183b4a1e77b984c01d17f3d5e734/pkg/networkservice/chains/nsmgr/heal_test.go#L180
And so it is only `10s` for the test. So we have `5s` (or less) for the request before healing and rest of the time for the requests after healing.

Reply:
ok, got it